.github/workflows/release-dev-from-beta/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Release Dev From Beta
+description: Dev release is created from the latest beta release.
+
+inputs:
+  BUILD_ZIP_FILE_PATH:
+    description: 'The path to the zip file.'
+    required: true
+  PLUGIN_NAME:
+    description: 'The name of the plugin.'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Replace Release Name In Plugin File With Dev
+      shell: bash
+      run: |
+            unzip -o ${{ inputs.BUILD_ZIP_FILE_PATH }} ${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php
```

Comment:
Extract `${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php` to a variable

Reply:
Done